misk-tailwind/src/main/kotlin/misk/tailwind/TailwindsHead.kt

```diff
@@ -1,50 +0,0 @@
-package misk.tailwind
-
-import kotlinx.html.TagConsumer
-import kotlinx.html.body
-import kotlinx.html.head
-import kotlinx.html.html
-import kotlinx.html.link
-import kotlinx.html.meta
-import kotlinx.html.script
-import kotlinx.html.title
-import misk.turbo.addHotwireHeadImports
-
-fun TagConsumer<*>.TailwindHtmlLayout(appRoot: String, title: String, playCdn: Boolean, headBlock: TagConsumer<*>.() -> Unit = {}, bodyBlock: TagConsumer<*>.() -> Unit) {
```

Comment:
This file is a duplicate and was overriding the correct file in the classpath, likely a refactor gone wrong.